pages/common/convert.md

```diff
@@ -17,3 +17,7 @@
 - Horizontally append images:
 
 `convert {{image1.png}} {{image2.png}} {{image3.png}} +append {{image123.png}}`
+
+- Create gif from series of images:
```

Comment:
Create a gif from a series of images with a given delay in (unit?) between them.

Reply:
Thanks for the comments! :)